src/objectExplorer/objectExplorerUtils.ts

```diff
@@ -26,8 +27,13 @@ export class ObjectExplorerUtils {
         }
     }
 
-    public static getNodeUri(node: TreeNodeInfo): string {
-        const profile = <IConnectionProfile>node.connectionInfo;
+    public static getNodeUri(node: TreeNodeInfo | ConnectTreeNode): string {
```

Comment:
I would still suggest combining these into a single type union so that if we add other types in the future (or modify existing ones) we have less places to update and can find the places these are used easier.

Reply:
ok, I will look into doing that